### DISCUSSION TITLE
Add editor fallback

### DISCUSSION THREAD

**Discussion by zt64** [knqyf263/pet#309] [Ideas]
Hi, with the configure command it tries vim and then sensible-editor. However the aur package has been out of date for a while now and having to install multiple additional packages just to edit for this one program is a little much. Could reading from the `EDITOR` env var be added as the last fallback, if all else fails?

**Comment by RamiAwar** [comment]
This is already how the config Editor is supposed to work! If none is provided, it falls back to the EDITOR env var!

https://github.com/knqyf263/pet/blob/e356762320793655ae51c9a3276f295c265c5c02/config/config.go#L128

**Reply by zt64** [reply]
Oh. Interesting, I'm not sure why or what was different way back when I made this post, thanks anyways
